google-cloud-spanner/src/test/java/com/google/cloud/spanner/spi/v1/SpannerMetadataProviderTest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.spanner.spi.v1;
+
+import static org.junit.Assert.assertEquals;
+
+import com.google.common.collect.ImmutableMap;
+import io.grpc.Metadata;
+import io.grpc.Metadata.Key;
+import java.util.Map;
+import org.junit.Test;
+
+public class SpannerMetadataProviderTest {
+  @Test
+  public void testGetHeadersAsMetadata() {
+    Map<String, String> headers = ImmutableMap.of("header1", "value1", "header2", "value2");
+    SpannerMetadataProvider headerProvider = SpannerMetadataProvider.create(headers, "header3");
```

Comment:
rename the variable to match the type (`metadataProvider`)

Reply:
Done